app/code/Magento/Wishlist/Api/data/WishlistInterface.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+namespace Magento\Wishlist\Api\data;
+
+interface WishlistInterface
+{
+    /**#@+
+     * Constants defined for keys of  data array
+     */
+    const NAME = 'name';
+
+    const SHARING_CODE = 'sharing_code';
+
+    const CUSTOMER_ID = 'customer_id';
+
+    const SHARED = 'shared';
+
+    const UPDATED_AT = 'updated_at';
```

Comment:
This field seems to be redundant. Do you have the specific case?

Reply:
Should not all fields used by getter and setter methods be specified in api constants?